src/classes/FORM_ServiceGiftEntry_TEST.cls

```diff
@@ -89,12 +89,11 @@ public with sharing class FORM_ServiceGiftEntry_TEST {
 
         FORM_Template template = UTIL_UnitTestData_TEST.createSampleTemplate();
         Gift_Entry_Settings__c giftEntryCustomSetting = new Gift_Entry_Settings__c();
-        giftEntryCustomSetting.Default_Single_Gift_Entry_Template__c = template.name;
-
-        UTIL_CustomSettingsFacade.getGiftEntrySettingsForTests(giftEntryCustomSetting);
 
         String templateJSON = JSON.serialize(template);
-        fs.storeFormTemplate(null, template.name, template.description, template.version, templateJSON);
+        String storedFormTemplateId = FORM_ServiceGiftEntry.storeFormTemplate(null, template.name, template.description, template.version, templateJSON);
```

Comment:
@dfuller114 I had to update your test class to work with the new changes.

cc: @vmartinov-sforg